electron.html

```diff
@@ -364,11 +364,9 @@
 
 var urlParams = new URLSearchParams(window.location.search);
 
-if (navigator.userAgent.indexOf('Mac OS X') != -1){
-	document.getElementById("warning4mac").style.display="block";
-} else if (location.hostname.toLowerCase() == "obs.ninja"){
+if ((location.hostname.toLowerCase() == "vdo.ninja") || (location.hostname.toLowerCase() == "obs.ninja")){
```

Comment:
Using three equal sings will make this more reliable

```suggestion
if ((location.hostname.toLowerCase() === "vdo.ninja") || (location.hostname.toLowerCase() === "obs.ninja")){
```